code/game/gamemodes/cult/runes.dm

```diff
@@ -656,6 +656,9 @@
 		D.visible_message("<span class='warning'>[D] slowly dissipates into dust and bones.</span>", \
 		"<span class='warning'>You feel pain, as bonds formed between your soul and this homunculus break.</span>", \
 		"<span class='warning'>You hear faint rustle.</span>")
+		if(D.mind in cult)
```

Comment:
This isn't the right place for this. It should be in a destroy proc for either the mind or the manifested ghost.